.travis.yml

```diff
@@ -37,7 +37,10 @@ before_script:
   - echo "zend_extension=xdebug.so" >> ~/.phpenv/versions/$(phpenv version-name)/etc/php.ini
 
 script:
-  - vendor/bin/phpunit -c phpunit.xml.dist --coverage-clover build/logs/clover.xml
+  - vendor/bin/phpunit -c phpunit.xml
```

Comment:
I've just check this script configuration can be totally removed

See exemple 5.2 https://phpunit.de/manual/current/en/organizing-tests.html